evaluator.go

```diff
@@ -73,12 +78,24 @@ func (s *State) Clusters() map[string]Cluster {
 	return s.clusters
 }
 
+// enable clusterQueryResultCaching
```

Comment:
let's leave out for now (always enabled) and consider adding in if anyone requests it